wgpu-core/src/command/transfer.rs

```diff
@@ -43,19 +43,31 @@ impl TextureCopyView {
     fn to_selector(&self, aspects: hal::format::Aspects) -> hal::image::SubresourceRange {
         let level = self.mip_level as hal::image::Level;
         let layer = self.array_layer as hal::image::Layer;
-        hal::image::SubresourceRange {
-            aspects,
-            levels: level .. level + 1,
-            layers: layer .. layer + 1,
+
+        // TODO: Can't satisfy clippy here unless we modify
+        // `hal::image::SubresourceRange` in gfx to use `std::ops::RangeBounds`.
+        #[allow(clippy::range_plus_one)]
```

Comment:
should we put it on the whole function?

Reply:
Hmmm, I usually prefer to have `allow`s as tight as possible, and ideally documented, but I'll defer to whatever you pick :)